### PR TITLE
Compare dendrite main and hnt dendrite fork

### PR DESCRIPTION
**Pull request by tak-hntlabs**
Compare the two branches for rebase.

### CODE REVIEW

File: zion/README.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+# Purpose
+
+Additional packaages added for the Zion project, nothing in here should be in the Matrix Dendrite upstream, nor in the herenotthere/dendrite-fork.
```
Comment (texuf) [inline zion/README.md:3]: Spelling - packaages

File: zion/contracts/goerli/addresses/council.json [hunk 2]
```diff
@@ -0,0 +1 @@
+{"councilnft": "0xae599a7c5f2cad3acd773e1931045384d70cb4e6"}
```
Comment (texuf) [inline zion/contracts/goerli/addresses/council.json:1]: Should these coming in from the dendrite update? Won’t we be running a script in this repo to update them?

**Review by texuf — APPROVED** [review]

File: zion/contracts/goerli/addresses/council.json (hunk 2 above)
Comment (tak-hntlabs) [reply zion/contracts/goerli/addresses/council.json:1]: The script runs in the mono repo as part of the contract publish step. We don't have a way to publish them from mono repo to the dendrite fork. So I'm manually copying them over.

File: zion/README.md (hunk 1 above)
Comment (tak-hntlabs) [reply zion/README.md]: fixed

**Pull request closed without merging**